src/app/InteractionModelEngine.cpp

```diff
@@ -1408,5 +1408,32 @@ size_t InteractionModelEngine::GetNumDirtySubscriptions() const
     return numDirtySubscriptions;
 }
 
+void InteractionModelEngine::OnFabricRemoved(FabricIndex aFabricIndex)
```

Comment:
Where is similar logic for shutting down active `ReadClient`s?

Reply:
added, thanks